### ISSUE TITLE
United Kingdom Output Formatting

### ISSUE THREAD

**Issue by pnlrogue1** [chubin/wttr.in#343]
I know we're really awkward in the UK but we use Metric units for temperature and imperial for wind speed (so Centigrade and MPH). By default, wttr.in using Metric units for UK (understandable as we're in EMEA) but there's no way to force it into the correct combination.

Could we please either get a way of specifying the Wind and Tempareture units or just specifying the UK formatting?

Thank you!

**Comment by liyishuai** [comment]
This issue is more than UK: In Chinese we use Beaufort scale (proposed in #231) for wind speed. It'll be nice to have default formatting per language.

**Comment by chubin** [comment]
@pnlrogue1 We have one option for wind (`?M` for m/s; and by default it is km/h),
but we need some option for MPH too. Maybe `?i` for m*i*les?

@liyishuai I am not sure that it is a good idea to bind to the languages (at least, not for all languages; for example, it would not work for English). But we definitely need different output formats for wind.
Maybe `?b` for Beaufort scale? (#231)

**Comment by liyishuai** [comment]
Indeed, I should request "default formatting per locale" rather than "per language".
`?b` for Beaufort looks good.

**Comment by chubin** [comment]
The problem is, that the local is not passed by default by the browser; and for the multi-language versions like zh.wttr.in, we have in the server name only the language name, and not the country name.

`?b` for Beafort accepted.
